remote_execution/oss/re_grpc/src/client.rs

```diff
@@ -762,11 +762,21 @@ impl REClient {
 
     pub async fn upload_blob(
```

Comment:
make it match internal RE signature:
```suggestion
     pub async fn upload_blob_with_digest(
/
        &self,
        blob: Vec<u8>,
        digest: TDigest,
        metadata: RemoteExecutionMetadata,
    ) -> anyhow::Result<TDigest> {
        let blob = InlinedBlobWithDigest{digest:digest.clone(), blob, ..Default::default()};
        self.upload(
            metadata,
            UploadRequest {
                inlined_blobs_with_digest: Some(vec![blob]),
                files_with_digest: None,
                directories: None,
                upload_only_missing: false,
                ..Default::default()
            },
        )
        .await?;
        Ok(digest)
    }
```